src/main/java/thewaypointers/trafficsimulator/TrafficSimulatorManager.java

```diff
@@ -1,76 +0,0 @@
-package thewaypointers.trafficsimulator;
-
-import thewaypointers.trafficsimulator.common.IStateProvider;
-import thewaypointers.trafficsimulator.common.WorldStateDTO;
-import thewaypointers.trafficsimulator.common.helpers.JunctionTestProvider;
-import thewaypointers.trafficsimulator.gui.MainFrame;
-
-import javax.swing.*;
-
-public class TrafficSimulatorManager extends Thread {
```

Comment:
This got renamed to StateProviderController, as it's an "engine" for a state provider.